snuba/query/allocation_policies/per_referrer.py

```diff
@@ -63,6 +63,18 @@ def _additional_config_definitions(self) -> list[AllocationPolicyConfig]:
                 value_type=int,
                 default=-1,
             ),
+            AllocationPolicyConfig(
+                name="throttle_threshold",
+                description="The threshold at which we will decrease the number of threads (THROTTLED_THREADS) used to execute queries",
+                value_type=int,
+                default=50,
```

Comment:
avoid magic numbers, I know there's already one above here so let's fix that too